README.md

```diff
@@ -367,9 +378,76 @@ Slurm specific parameters that do not need to be modified by non-expert users.
 **only_generate_features**
 If set to True, stops after generating features and does not perform structure prediction and reporting.
 
+
 ## 3. Execution
 
-After following the Installation and Configuration steps, you are now ready to run the snakemake pipeline. To do so, navigate into the cloned pipeline directory and run:
+After following the Installation and Configuration steps, you are now ready to run the Snakemake pipeline. To do so, navigate into the cloned pipeline directory and run:
+
+```bash
+snakemake \
+  --use-singularity \
+  --singularity-args "-B /scratch:/scratch \
+    -B /g/kosinski:/g/kosinski \
+    --nv " \
+  --jobs 200 \
+  --restart-times 5 \
+  --profile slurm_noSidecar \
+  --rerun-incomplete \
+  --rerun-triggers mtime \
+  --latency-wait 30 \
+  -n
+```
+
+> [!Warning]
+> Running Snakemake in the foreground on a remote server can cause the process to terminate if the session is disconnected. To avoid this, you can run Snakemake in the background and redirect the output to log files. Here are two approaches depending on your environment:
+
+- **For SLURM clusters:** Use `srun` to submit the job in the background:
+
+  ```bash
+  srun --job-name=snakemake_job \
+    snakemake \
+    --use-singularity \
+    --singularity-args "-B /scratch:/scratch \
+      -B /g/kosinski:/g/kosinski \
+      --nv " \
+    --jobs 200 \
+    --restart-times 5 \
+    --profile slurm_noSidecar \
+    --rerun-incomplete \
+    --rerun-triggers mtime \
+    --latency-wait 30 \
```

Comment:
maybe add &>log.txt& to redirect all messages (not all of them are in the .snakamake log dir)